Documentation/installing.md

```diff
@@ -0,0 +1,185 @@
+
+Installations from the zip file are self-contained, so in a sense work like a Python virtual environment. For PowerShell users, the zip archive also contains script `Enter-IronPythonEnvironment.ps1` that works similarly to Anaconda's `Enter-CondaEnvironment.ps1` or CPython's `activate.ps1`. Use `help` on this file for detailed usage information.
+
+#### _Manual_
+
+Download and unzip `IronPython.3.X.Y.zip` (`X` and `Y` being the minor and patch version numbers). The unzipped directory contains several .NET Runtime version-specific subdirectories and a subdirectory `lib`. Pick a subdirectory that matches the .NET Runtime version installed on your system (or lower), and **move** subdirectory `lib` into that directory. IronPython can then be launched with
+
+```
+dotnet /path/to/unzipped/netXXX/ipy.dll
+```
+
+#### _Scripted_
+
+The zip file contains a helper PowerShell script to install IronPython properly in a designated location and creates a launcher script. In this way one zip file can "seed" multiple virtual environments, each starting "blank" that is, with only the Python Standard Library present.
+
+```
+/path/to/unzipped/scripts/Install-IronPython.ps1 -Path ~/ipyenv -ZipFile ~/Downloads/IronPython.3.X.Y.zip
+```
+
+Use Powershell's `help` command on the script for information about available options and switches.
+
+The script is also available online, so it can be downloaded and invoked without unzipping the archive first.
+
+```
+PS> Invoke-WebRequest https://raw.githubusercontent.com/IronLanguages/ironpython3/master/Src/Scripts/Install-IronPython.ps1 -OutFile ./Install-IronPython.ps1
+PS> ./Install-IronPython ~/ipyenv ~/Downloads/IronPython.3.X.Y.zip
+PS> ~/ipyenv/Enter-IronPythonEnvironment
+«ipyenv» PS> ipy
+IronPython 3.4.0 (3.4.0.1000)
+[.NETCoreApp,Version=v6.0 on .NET 6.0.12 (64-bit)] on win32
+Type "help", "copyright", "credits" or "license" for more information.
+>>>
+```
+
+### Standalone (TODO)
+
+**NOTE**: This section is tentative. None of the installation options mentioned here are supported yet.
+
+A standalone installation does not require any prior .NET Runtime.
+
+#### _Windows_
+
+Use Chocolatey (?)
+
+#### _Linux_
+
+Use Flatpak (?) or Snap (?)
+
+#### _macOS_
+
+Install Homebrew cask (?)
+
+## IronPython on .NET Framework/Mono
+
+### Windows
+
+IronPython for .NET Framework requires .NET Framework version 4.6.2 or higher, which comes preinstalled on modern versions of Windows. To install IronPython, download the `.msi` file from the project's [release page](https://github.com/IronLanguages/ironpython3/releases/latest) and execute it. Alternatively, use _Chocolatey_ package manager:
+
+```
+choco install ironpython
+```
+
+The installation registers IronPython in the System Registry in compliance with [PEP 514](https://peps.python.org/pep-0514/) so that other tools may detect and use it.
```

Comment:
I think the chocolatey installer doesn't do this...

Reply:
Oh, that's too bad. Can it be done?

For now I'll reorder the lines so that this note comes before Chocolatey.

Reply:
I also wonder how this would interact/conflict with a parallel installation though a `.msi`…